db/osqlcomm.c

```diff
@@ -5046,11 +5046,11 @@ void *osql_create_request(const char *sql, int sqlen, int type,
         req_uuid.rqlen = rqlen;
         req_uuid.sqlqlen = sqlen;
 
-        if (tzname)
-            strncpy0(r_uuid_ptr->tzname, tzname, sizeof(r_uuid_ptr->tzname));
```

Comment:
We are preparing the payload via req_uuid, so better copy tzname to req_uuid and let osqlcomm_xxx_put() copy it to the buffer? Ditto for non-uuid case.